hooks/qamp-projects-conversion-utils.ts

```diff
@@ -0,0 +1,126 @@
+import Airtable from "airtable";
+import { QampCohort, QampParticipant, QampProject } from "../types/qamp";
+import { AirtableRecords } from "./airtable-conversion-utils";
+import QampCohortsAirtableRecords from "./qamp-cohorts-conversion-utils";
+import QampParticipantsAirtableRecords from "./qamp-participants-conversion-utils";
+
+const RECORD_FIELDS_IDS = Object.freeze({
+  cohort: "fldaIxSsaydaOCUGK",
+  collaborators: "fldaoHhT2ttO35K7T",
+  githubIssueUrl: "fldrZHw1f1nQwO9VX",
+  mentees: "flda8motpH4gS5bns",
+  mentors: "fldgBeYDL1YKSpkXy",
+  name: "fldtLoW4vLVGzsGXa",
+} as const);
+
+const AIRTABLE_BASE_ID = "appjU5TOUDYPwBIqO";
+
+class QampProjectsAirtableRecords extends AirtableRecords {
+  protected qampCohorts: QampCohort[] = [];
+  protected qampParticipants: QampParticipant[] = [];
+
+  constructor(
+    apiKey: string,
+    recordFields?: Record<string, any>,
+    qampCohorts?: QampCohort[],
+    qampParticipants?: QampParticipant[]
+  ) {
+    super(
+      apiKey,
+      AIRTABLE_BASE_ID,
+      "Projects",
+      "Grid view",
+      undefined,
+      recordFields
+    );
+    this.qampCohorts = qampCohorts || [];
+    this.qampParticipants = qampParticipants || [];
+  }
+
+  async fetchProjects(): Promise<QampProject[]> {
+    if (!this.recordFields) {
+      this.recordFields = await this.getAllFieldNames(RECORD_FIELDS_IDS);
+    }
+
+    // Fetch QAMP cohorts
```

Comment:
I don't think this comment adds new information, the variable name is really clear